evaluator/remediations.go

```diff
@@ -0,0 +1,28 @@
+package evaluator
+
+import (
+	"app/base"
+	"app/base/mqueue"
+	"github.com/RedHatInsights/patchman-clients/vmaas"
+)
+
+var remediationsPublisher = mqueue.WriterFromEnv("REMEDIATIONS_UPDATE_TOPIC")
+
+type RemediationsState struct {
+	HostID string   `json:"host_id"`
+	Issues []string `json:"issues"`
+}
+
+func publishRemediationsState(id string, response vmaas.UpdatesV2Response) error {
+	var state RemediationsState
+	state.HostID = id
+	state.Issues = make([]string, 0)
```

Comment:
We call `getReportedAdvisories` twice during evaluation it would be better to reuse its result, but I see it would make our code more complex. So at least we can create "reporetedAdvisories" first and then preallocate state.Issues with the exact length.